### ISSUE TITLE
[Request] Allow portfolio and selector models to be set through hyperparameters in ASKL2

### ISSUE THREAD

**Issue by PGijsbers** [automl/auto-sklearn#1197]
As per the title, it will be useful for ASKL2 to have a configurable `portfolio` and `policy selector`.
It's beneficial for research (avoiding 'cheating' through meta-learning in a benchmark) or for customization.

Issue opened on the request of @mfeurer

**Comment by eddiebergman** [comment]
Hi @PGijsbers, this would certainly be a nice feature and definitely something to have in once asklearn 2 becomes non-experimental. I had a quick look to see how possible this is and I don't imagine it's too bad depending on the definition of configurable for `policy selector` and `portifolio`.

Simply changing the `portifolio` file location and enabling disabling it should be quite straightforward and likewise with white listing or black listing certain policys. However if there is a need for injecting a custom `policy selector` object it might be a bit more involved ... but also manageable.

Do you need this for the `automlbenchmark` and do you have a time urgency for this?

**Comment by PGijsbers** [comment]
Yes it's something we'd ultimately like to use in the AutoML benchmark. Though realistically I don't think we can wait for this to be all done and set up properly before doing our next set of experiments, which have been delayed for months already.

**Comment by eddiebergman** [comment]
No problem, seeing as this will probably not be done in time for the benchmarks, I will talk to @mfeurer about how we do this. Perhaps autosklearn2 will become the default before the next benchmark run.

**Comment by zhuygln** [comment]
Hi, 
We have been using autosklearn and automlbenchmark in R&D at SAS Institiute for autoML pipeline. Huge applauses to the great work of you all first! @eddiebergman @PGijsbers 

I have a few questions regarding portfolio. 

1.  I am trying to replicate the portfolio building in the autosklearn2 paper. Is these any code shared in this repo? I search the portfolio file but I couldn't find it. Maybe I have don't have the permission to the version that you are referring to. 
2.  For my current research project, I'd like to add/delete portfolio. Is this supported? If not, I am glad to contribute by adding this feature. 

Thanks,
Yonglin
